lib/migration.pm

```diff
@@ -118,6 +120,15 @@ sub remove_ltss {
         }
         set_var('SCC_ADDONS', join(',', grep { $_ ne 'ltss' } @$scc_addons));
     }
+    if (get_var('DROPPED_MODULES', '')) {
```

Comment:
I think you can combine ltss and DROPPED_MODULES in a list, then deal each one in a loop.

Reply:
Yes, I modified the code as we discussed, will see the verification run results, thanks.